Reworking Azure Developer Hub for consistency with other languages (#18926)
dotnet/docs: docs/azure/intro.md

**Managed services**: Azure provides some services where you provide data or information to Azure, and Azure's implementation takes the appropriate action. One example is Azure Blob Storage, where you provide files and Azure handles reading, writing, and persisting them.

> **Azure SDK for .NET**: Sometimes referred to as the **Azure libraries for .NET**, this collectively refers to the [NuGet packages](https://www.nuget.org/profiles/azure-sdk) you can install in your project to provide connectivity to various Azure resources. These packages also include management libraries used to provision and administer the resources.

Reviewer: ```suggestion
**Azure SDK for .NET**: Sometimes referred to as the **Azure libraries for .NET**, this collectively refers to the [NuGet packages](https://www.nuget.org/profiles/azure-sdk) you install in your project that provide various interactions and functionality with Azure. These packages also include management libraries used to provision and administer the resources.
```

Reviewer 2: I have to say "functionality with Azure" reads awkward to me.

> 

Author: @IEvangelist What do you think about adding this definition here?

Reviewer: I made a suggestion

If you use Visual Studio, you can add support for certain Azure services to your projects. The **Connected Services** dialog in Visual Studio provides an easy way to add all the required references, connection code, and configuration settings to your projects. Some commonly used Azure services are supported out of the box, such as [Storage](/azure/vs-azure-tools-connected-services-storage), [Azure Active Directory](/azure/active-directory/develop/vs-active-directory-add-connected-service) authentication, [Azure Key Vault](/azure/key-vault/vs-key-vault-add-connected-service), and [Cognitive Services](/azure/cognitive-services/) such as [Computer Vision](/azure/cognitive-services/computer-vision/vs-computer-vision-connected-service). More services, including third-party services, are available as extensions in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/search?term=connected%20service&target=VS&category=Tools&vsVersion=&subCategory=All&sortBy=Relevance).

## Using the Azure SDK for .NET

If you're using the Azure SDK for .NET to consume or manage your Azure resources, please note the following:

> * **Authentication**: Many libraries in the SDK use a common authentication infrastructure, while some libraries use authentication mechanisms specific to the service they're consuming. See [Authentication with the Azure SDK for .NET](authentication.md) for further details.

Reviewer: ```suggestion
* **Authentication**: Many libraries in the SDK use a common authentication infrastructure, while some libraries use authentication mechanisms specific to the service they're consuming. For more information, see [Authentication with the Azure SDK for .NET](authentication.md).
```

> * **Authentication**: Many libraries in the SDK use a common authentication infrastructure, while some libraries use authentication mechanisms specific to the service they're consuming. See [Authentication with the Azure SDK for .NET](authentication.md) for further details.

Reviewer 2: I don't think libraries are "consuming" services. I would say libraries provide access to services.

Author: Agreed

> * **Logging**: If supported, the client libraries include the ability to log client library operations. See [Logging with the Azure SDK for .NET](logging.md) for further details.

Reviewer: ```suggestion
* **Logging**: If supported, the client libraries include the ability to log client library operations. For more information, see [Logging with the Azure SDK for .NET](logging.md).
```

* **REST API** The Azure SDK for .NET is an abstraction built on the [Azure REST API](https://docs.microsoft.com/rest/api/azure/). The Azure REST API may be used in lieu of or alongside the Azure SDK for .NET if desired.

> 

Author: @IEvangelist and also this
